### PR TITLE
Server type update

### LINKED ISSUE #103: Fix configuration option in create server endpoint
Type options are not displayed in config. Add enums for jupyter, restful, and cron.

### PR DESCRIPTION
**Pull request by jgwerner**
Closes #103

### CODE REVIEW

**Comment by jgwerner** [comment]
@jkp85 @johngriebel could you guys double check this? I'm not sure how to specify the config --> `type: jupyter/restful/cron` K/V with enums.

File: tbs_swagger.yml [hunk 1]
```diff
@@ -4821,8 +4821,10 @@ definitions:
         type: string
       interval:
         description: 'Plan interval.'
-        type: string
-        enum: [day, week, month, year]
+        type: array
```
Comment (johngriebel) [inline tbs_swagger.yml]: I'm not sure why the type should be `array` when a `string` really is what is sent to the API. I assume it's the only way to get the `enum` attribute to work correctly?

Comment (jgwerner) [inline tbs_swagger.yml]: In this case it's a mistake. The parent is `interval` and the enums should be strings.

Comment (jgwerner) [inline tbs_swagger.yml]: @johngriebel corection, an array of strings is correct. This allows swagger ui's to display a list of possible enums with day, week, month, year.

**Comment by jgwerner** [comment]
@jkp85 @johngriebel unless you have any objections, i will go ahead and merge.

**Pull request merged**